### PR TITLE
add Onramp Money to the dapp list

### PR DESCRIPTION
**Pull request by sajwanAshish**
> To make a submission to the Dapps Page, please complete the task lists
> below; otherwise remove them.

Your Dapp:

- [x] connects to Valora via WalletConnect V2;
- [x] must be mobile-first or mobile responsive;
- [x] is in one of the existing categories; and
- [x] includes a link to user documentation.

For more information see the [Valora Listing Overview](https://docs.valora.xyz/listing/overview).

### CODE REVIEW

File: locales/base.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "immortalx": "Trade crypto with low fees directly from your wallet, with up to 50x leverage",
     "sarafu": "Create, use and explore Community Asset Vouchers with Grassroots Economics",
     "hedgey": "Empower your team and investors with on-chain token vesting",
-    "talentprotocol": "Where builders commit to goals, share their progress and find the support they need to succeed"
+    "talentprotocol": "Where builders commit to goals, share their progress and find the support they need to succeed",
+    "onrampmoney": "The financial platform for buying/selling crypto and accepting crypto payments.Convert fiat to web3 money"
```
Comment (MuckT) [inline locales/base.json:42]: The description should be 50 characters or less and adhere to the other standards listed - [description requirements](https://docs.valora.xyz/listing/adding-new-dapps#required-changes).

**Comment by sajwanAshish** [comment]
@MuckT can you please have a look at this. Looks like some checks are failing.

**Comment by MuckT** [comment]
@sajwanAshish when attempting to get a OTP from SMS I am getting the following message: 
`OTP not sent FirebaseError: Firebase: Error (auth/too-many-requests).`

**Comment by sajwanAshish** [comment]
> @sajwanAshish when attempting to get a OTP from SMS I am getting the following message: `OTP not sent FirebaseError: Firebase: Error (auth/too-many-requests).`

- please try the telegram login option

**Comment by MuckT** [comment]
@sajwanAshish It appears that the OTP SMS is WAI intended now. Is there sample test data that can be used to test the flow end to end?

**Comment by sajwanAshish** [comment]
> @sajwanAshish It appears that the OTP SMS is WAI intended now. Is there sample test data that can be used to test the flow end to end?

**Comment by sajwanAshish** [comment]
> > @sajwanAshish It appears that the OTP SMS is WAI intended now. Is there sample test data that can be used to test the flow end to end?

Phone: +971 999374982
OTP: 123456
https://onramp.money/app?appId=2

**Comment by sajwanAshish** [comment]
> > > @sajwanAshish It appears that the OTP SMS is WAI intended now. Is there sample test data that can be used to test the flow end to end?
> 
> Phone: +90 9993749853 OTP: 123456 Sandbox URL: https://onramp.money/app?appId=2
> 
> Test the flow here

this is dummy user

**Review by MuckT — APPROVED** [review]

**Pull request merged**